docling_ibm_models/code_formula_model/models/sam.py

```diff
@@ -0,0 +1,502 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+
+# This source code is licensed under the license found in the
+# LICENSE file in the root directory of this source tree.
```

Comment:
This file header is not correct in the context of this package. Please clean this up and add proper attribution on the root of docling-ibm-models.